web-app/package.json

```diff
@@ -25,17 +24,25 @@
     "google-protobuf": "^3.6.1",
     "grpc-web": "^1.0.0",
     "js-cookie": "^2.2.1",
-    "polka": "next",
-    "sirv": "^0.4.0"
+    "polka": "next"
   },
   "devDependencies": {
     "browserify": "^16.2.2",
```

Comment:
Were you able to tell why we need both Browserify and Webpack?

Reply:
I threw it in there blindly when I first added the proto build steps (following the docs...I don't really know the difference between the two), but the app still builds with Browserify removed, so will drop it from the deps.